kodi_addon_checker/__main__.py

```diff
@@ -65,6 +65,7 @@ def main():
     parser.add_argument("dir", type=dir_type, nargs="*", help="optional add-on or repo directories")
     parser.add_argument("--branch", choices=choice, required=True,
                         help="Target branch name where the checker will resolve dependencies")
+    parser.add_argument("--PR", help="Tell if tool is to run on a pull requests or not", action='store_true')
```

Comment:
maybe lowercase `--PR`
@Razzeee what's your opinion?

Reply:
I would prefer uppercase. But I'm curious if it even matters, maybe the argument works case insensitive anyway?

Reply:
No it's case sensitive
It's fine for me to keep it uppercase

Reply:
It's not case insensitive. it needs to match the used case.